src/event/event_internal.h

```diff
@@ -125,6 +125,8 @@ _dispatch_timer_flags_from_clock(dispatch_clock_t clock)
 
 #if defined(_WIN32)
 typedef uintptr_t dispatch_unote_ident_t;
+#elif defined(__OpenBSD__)
+typedef uint32_t dispatch_unote_ident_t;
```

Comment:
Unless the typedef is going to change, that is an unnecessary ifdef, no?

Reply:
I was experimenting a little here and forgot to change this back. This needs to match `struct kevent`, so should be a `uintptr_t`, and necessitates some further changes.